templates/legal/anbox-cloud-terms-of-service.html

```diff
@@ -0,0 +1,92 @@
+        <p>Canonical may disclose any or all personal data and contents you have sent, posted or published if required to comply with applicable law or the order or requirement of a court, administrative agency or other governmental body. All other use of your personal data is subject to the Privacy Policy.</p>
+      </section>
+      <section>
+        <h2 id="liability">9. Liability</h2>
+        <p>Your use of the Service is at your sole risk. The Service is provided on an “as is” and “as available” basis without warranty of any kind.</p>
+        <p>In the case of the Service provided by Canonical, the Service is provided “as is” and, other than as expressly set out in these Terms of Service, all warranties (whether express, implied, statutory or otherwise) in respect of the Services are expressly excluded to the maximum extent permitted by law.</p>
+        <p>Canonical will provide the Services with reasonable care and skill, and Canonical will use reasonable efforts to ensure the availability of the Services, but makes no guarantee that the Services will be available without interruption or will be error-free.</p>
+        <p>Canonical will not be liable in contract, tort or otherwise for any: indirect or consequential loss; loss of profits; loss of revenue; loss of anticipated savings; loss of business or business opportunity; loss of goodwill; or loss of or corruption to data. Otherwise, Canonical’s total liability in contract, tort or otherwise for any claims is limited to £10.</p>
+        <p>Nothing in these Terms of Service will exclude or limit Canonical’s liability for: death or personal injury caused by the negligence of Canonical; fraud or fraudulent misrepresentation; or any other liability that cannot be excluded or limited by law.</p>
+      </section>
+      <section>
+        <h2 id="general">9. General</h2>
+        <p>These Terms of Service are governed by the laws of England and any dispute will be heard by the courts in England.</p>
+        <p>Failure by Canonical to enforce any right or provision of these Terms of Service shall not constitute a waiver of such right or provision. If any part of these Terms of Service is held invalid or unenforceable, that part will be construed to reflect the parties original intent, and the remaining portions will remain in full force and effect. The terms of these Terms of Service do not affect your statutory rights.</p>
+        <p>Any notices should be sent by email to <a href="mailto:legal@canonical.com">legal@canonical.com</a> or by registered post to:</p>
+        <p>Canonical Group Limited,<br>
+        5th Floor, Blue Fin Building,<br>
+        110 Southwark Street,<br>
+        London, SE1 0SU.</p>
```

Comment:
I know it's in the copy doc but the pedant in me is pretty sure the period isn't needed in this instance:

```suggestion
        London, SE1 0SU</p>
```